.vscode/snippets.code-snippets

```diff
@@ -0,0 +1,65 @@
+{
+  "Browser Compatibility Data Template": {
+    "scope": "json",
+    "prefix": ["bcd", "browser-compat"],
+    "body": [
+      "{",
+      "  \"${1|api,css,html,http,javascript,mathml,svg,webdriver,webextensions,xpath,xslt|}\": {",
+      "    \"${2:${TM_FILENAME_BASE}}\": {",
+      "      \"__compat\": {",
+      "        \"mdn_url\": \"${3:mdn-url}\",",
+      "        \"support\": {",
+      "          \"chrome\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"chrome_android\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"edge\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"edge_mobile\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"firefox\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"firefox_android\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"ie\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"opera\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"opera_android\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"safari\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"safari_ios\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"samsunginternet_android\": {",
+      "            \"version_added\": null",
+      "          },",
+      "          \"webview_android\": {",
+      "            \"version_added\": null",
+      "          }",
+      "        },",
+      "        \"status\": {",
+      "          \"experimental\": ${4|true,false|},",
+      "          \"standard_track\": ${5|true,false|},",
+      "          \"deprecated\": ${6|true,false|}",
```

Comment:
Can we make this ` ${6|false,true|}"` ? 
If you don't tab through until here, it is set to `true` by default and `deprecated: true` isn't typical most of the time.